listenbrainz/webserver/static/css/listens-page.less

```diff
@@ -310,7 +310,12 @@
 				padding-right:initial;
 			}
 		}
-		
+
+		.title-duration {
+			display: flex;
+			flex-direction: row;
+			column-gap: 0.5rem;
```

Comment:
Just in case, a translation of the less/css:
`>` :  Target direct children of title-duration
`*`: Target any element type (that way if we change the element type one day we won't be breaking everything)
`:not(…)`: pretty straightforward, inverts whatever condition is in the parens
`:first-child`: no explanation needed :)

All in all this translates to "target any and every child of title-duration except for the first one"